src/components/BulkEditList/BulkEditListFilters/utils/getFileInfo.js

```diff
@@ -0,0 +1,11 @@
+import { getFileExtension } from '@folio/stripes-data-transfer-components';
+import { SUPPORTED_FILE_EXTENSIONS } from '../../../../constants/constants';
+
+export const getFileInfo = file => {
+  const fileType = getFileExtension(file).slice(1);
+
+  return {
+    isTypeSupported: Boolean(SUPPORTED_FILE_EXTENSIONS[fileType.toUpperCase()]),
```

Comment:
When SUPPORTED_FILE_EXTENSIONS will be array, we can just

const isTypeSupported =  `SUPPORTED_FILE_EXTENSIONS.includes(fileType)`

Reply:
Done.